web/beacon-app/src/features/home/components/QuickStart.tsx

```diff
@@ -1,9 +1,12 @@
+import OtterLookingDown from '@/components/icons/otter-looking-down';
+
 import SetupNewProject from '../components/SetupNewProject';
 import AccessDocumentationStep from './AccessDocumentationStep';
 
 export default function QuickStart() {
   return (
-    <div className="space-y-10">
+    <div className="relative w-full space-y-10">
+      <OtterLookingDown className="2xl:right-18 hidden md:absolute md:-top-[215px] md:left-8 md:block md:scale-[.65] xl:left-3/4" />
```

Comment:
I will suggest reviewing the position of the otter to be visible from medium (`md`) to the big screen(`2xl`) and on the same place